hedera-node/src/main/java/com/hedera/services/records/TransactionRecordService.java

```diff
@@ -144,4 +150,13 @@ public void includeOwnershipChanges(OwnershipTracker ownershipTracker) {
 		}
 		txnCtx.setTokenTransferLists(transferLists);
 	}
+
+	/**
+	 * This method saves contract results to {@link TransactionContext}.
```

Comment:
Normally in java docs it is starge to see "This method does X and Y". I think it would be better to stick to the general guidelines and change the java doc to 
`Externalises the transaction execution results to the {@link TransactionContext}`